lib/ramfs/README.md

```diff
@@ -0,0 +1,31 @@
+# RamFS: Unikraft’s RAM-based Virtual Filesystem
+
+The `RamFS` filesystem is a virtual filesystem that uses memory for storage, as the name implies. Its advantage is that it is very fast. From a simplified perspective, we can look at a file in RamFS as a buffer in memory. Since this is how RamFS represents data, the disadvantage becomes apparent: we can only write data into RamFS until we fill up all the memory.
```

Comment:
```suggestion
The `RamFS` filesystem is a virtual filesystem that uses memory for storage, as the name implies. Its advantage is that it is very fast. From a simplified perspective, we can look at a file in `RamFS` as a buffer in memory. Since this is how `RamFS` represents data, the disadvantage becomes apparent: we can only write data into `RamFS` until we fill up all the memory.
```